misc/sampleconfigs/gpio_settings.ini.pirate-audio-hat.sample

```diff
@@ -0,0 +1,34 @@
+[DEFAULT]
+enabled: True
+
+[PlayPause]
+enabled: True
+Type: Button
+Pin: 5
+pull_up: True
+bouncetime: 250
+functionCall: functionCallPlayerPause
+
+[VolumeUp]
+enabled: True
+Type:  Button
+Pin: 20
```

Comment:
Need also to add pin 24, see https://github.com/pimoroni/pirate-audio/issues/63#issuecomment-916860634